build_tools/github/test_windows_wheels.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+set -x
+
+PYTHON_VERSION=$1
+BITNESS=$2
+
+if [[ "$PYTHON_VERSION" == "36" || "$BITNESS" == "32" ]]; then
+    # For Python 3.6 and 32-bit architecture use the regular
+    # test command (outside of the minimal Docker container)
+    cp $CONFTEST_PATH $CONFTEST_NAME
+    pytest --pyargs sklearn
+    python -m threadpoolctl -i sklearn
+else
+    docker container run --name minimal_windows \
+                         -d -ti --rm scikit-learn/minimal-windows powershell
```

Comment:
I find this combination of docker run powershell  in detached interactive mode + subsequent docker exec commands a bit hackish. I hope that travis will clean up the detached interactive container.

Could you please at least explain why we did not use a single docker run command (in non interactive mode)? It's probably possible to make it work without detaching a powershell container but it would require a local windows install of docker to be able to experiment without suffering from  "try / wait for CI to fail / scroll the logs / tweak script / try again" slow iteration cycles.

Reply:
> I find this combination of docker run powershell in detached interactive mode + subsequent docker exec commands a bit hackish. I hope that travis will clean up the detached interactive container.

Yes, I agree that this is a bit hackish.

> Could you please at least explain why we did not use a single docker run command (in non interactive mode)? It's probably possible to make it work without detaching a powershell container but it would require a local windows install of docker to be able to experiment without suffering from "try / wait for CI to fail / scroll the logs / tweak script / try again" slow iteration cycles.

I did that because I was not able to make it work directly with `docker container run`. Nevertheless, I have remembered that using `powershell -Command "pytest --pyargs sklearn"` and `powershell -Command "python -m threadpoolctl -i sklearn"` would work for `docker container run` without a detached interactive container.

Reply:
Nice :)